packages/@pollyjs/core/src/adapters/fetch/index.js

```diff
@@ -68,12 +70,26 @@ export default class FetchAdapter extends Adapter {
   async respond(pollyRequest, status, headers, body) {
     await pollyRequest.respond(status, headers, body);
 
-    const { response } = pollyRequest;
+    const { url, response } = pollyRequest;
 
-    return new Response(response.body, {
+    const fetchResponse = new Response(response.body, {
       status: response.statusCode,
       headers: response.headers
     });
+
+    /*
+      Response does not allow `url` to be set manually (either via the
+      constructor or assignment) so force the url property via `defineProperty`.
+    */
+    defineProperty(fetchResponse, 'url', {
+      /*
+        Since `url` can be a relative url and Response always has an absolute
+        one, use URL to stitch together the origin if necessary.
```

Comment:
Unsure what stitch together the origin means

Reply:
Oh my bad, I meant that it attaches the host name to a relative url. I'll update the comment.